components/staging/staging.html

```diff
@@ -20,9 +20,17 @@
           </span>
           <span class="commit-message-title-counter" data-bind="text: commitMessageTitleCount"/>
         </div>
-        <button class="btn btn-primary btn-lg commit-btn" data-bind="click: commit, visible: isStageValid, enable: !commitValidationError()">
-          Commit
-        </button>
+        <div class="btn-group commit-grp">
+          <button class="btn btn-primary btn-lg commit-btn" data-bind="click: commit, visible: isStageValid, enable: !commitValidationError()">
```

Comment:
I think the `visible: isStageValid` should be on the `btn-group` because if there is a conflict the arrow of the dropdown is still visible and clickable.

![image](https://user-images.githubusercontent.com/4009570/36076348-39b67bb8-0f5b-11e8-9b88-422905f4139a.png)

Reply:
nice catch will fix it